deepmd/utils/tabulate.py

```diff
@@ -352,6 +421,10 @@ def _get_table_size(self):
                 table_size = self.ntypes
         elif isinstance(self.descrpt, deepmd.descriptor.DescrptSeT):
             table_size = int(comb(self.ntypes + 1, 2))
+        elif isinstance(self.descrpt, deepmd.descriptor.DescrptSeR):
+            table_size = self.ntypes * self.ntypes
+            if self.type_one_side :
+                table_size = self.ntypes
```

Comment:
Please add an `else` to raise the error of un-supported descritpor

Reply:
OK~